x-pack/test_serverless/functional/test_suites/search/common_configs/config.group1.ts

```diff
@@ -13,6 +13,7 @@ export default async function ({ readConfigFile }: FtrConfigProviderContext) {
   return {
     ...baseTestConfig.getAll(),
     testFiles: [
+      require.resolve('../../common/example'),
```

Comment:
Similar to my comment above. Suggest to also include this in observability and security tests.